conf/messages

```diff
@@ -3687,26 +3687,34 @@ hasEstimatedDetails.error.boolean=Select yes if any of the details in this retur
 
 hasEstimatedDetails.cyaChange=if any of the amounts in the return have been estimated
 
-yearToDateLiability.title=Tell us about your overall liability in tax year 2020 to 2021
-yearToDateLiability.agent.title=Tell us about your client’s overall liability in tax year 2020 to 2021
-yearToDateLiability.trust.title=Tell us about the trust’s overall liability in tax year 2020 to 2021
-yearToDateLiability.capacitor.title=Tell us about the person’s overall liability in tax year 2020 to 2021
-yearToDateLiability.personalRep.title=Tell us about the person’s overall liability in tax year 2020 to 2021
-yearToDateLiability.personalRepInPeriodOfAdmin.title=Tell us about the estate’s overall liability in tax year 2020 to 2021
-yearToDateLiability.personalRepInPeriodOfAdmin.agent.title=Tell us about your client’s overall liability in tax year 2020 to 2021
-
-yearToDateLiability.p1=To work out this amount, you’ll need to:
-yearToDateLiability.trust.p1=You’ll need to multiply the trust’s net gain by the correct <a href="{0}" target="_blank">rate of tax (opens in a new window)</a> to get this amount.
-yearToDateLiability.personalRepInPeriodOfAdmin.p1=You’ll need to multiply the estate’s net gain by the correct <a href="{0}" target="_blank">rate of tax (opens in a new window)</a> to get this amount.
-yearToDateLiability.personalRepInPeriodOfAdmin.agent.p1=You’ll need to multiply your client, or the estate’s net gain by the correct <a href="{0}" target="_blank">rate of tax (opens in a new window)</a> to get this amount.
-
-yearToDateLiability.li1=Multiply the amount of net gain to be taxed at the <a href="{0}" target="_blank">basic rate of tax (opens in a new window)</a> for the UK property-related disposal for that tax year.
-yearToDateLiability.li2=Multiply the amount to be taxed at the <a href="{0}" target="_blank">higher rate of tax (opens in a new window)</a> for property-related disposals for that tax year.
-
-yearToDateLiability.li3=Add these amounts together to get your overall Capital Gains Tax liability on your UK property-related disposals.
-yearToDateLiability.agent.li3=Add these amounts together to get your client’s overall Capital Gains Tax liability on their UK property-related disposals.
-yearToDateLiability.capacitor.li3=Add these amounts together to get the person’s overall Capital Gains Tax liability on their UK property-related disposals.
-yearToDateLiability.personalRep.li3=Add these amounts together to get the person’s overall Capital Gains Tax liability on their UK property-related disposals.
+yearToDateLiability.title=What is your overall Capital Gains Tax liability on your UK property-related disposals in tax year 2020 to 2021?
```

Comment:
to future-proof this, the tax year should be parameters passed into the message like the ones for `yearToDateLiability.h2`

Reply:
I just followed what was done before - they are all static:

```scala
yearToDateLiability.title=Tell us about your overall liability in tax year 2020 to 2021
yearToDateLiability.agent.title=Tell us about your client’s overall liability in tax year 2020 to 2021
yearToDateLiability.trust.title=Tell us about the trust’s overall liability in tax year 2020 to 2021
yearToDateLiability.capacitor.title=Tell us about the person’s overall liability in tax year 2020 to 2021
yearToDateLiability.personalRep.title=Tell us about the person’s overall liability in tax year 2020 to 2021
yearToDateLiability.personalRepInPeriodOfAdmin.title=Tell us about the estate’s overall liability in tax year 2020 to 2021
yearToDateLiability.personalRepInPeriodOfAdmin.agent.title=Tell us about your client’s overall liability in tax year 2020 to 2021
```

Reply:
That's really weird - the template actually passes in the tax year to the title but the messages were never set to accept them:

```scala
@title = @{messages(s"$key$userKey.title", taxYearStartYear, taxYearEndYear)}
```